tests/testthat/test-get_source_expressions.R

```diff
@@ -122,3 +122,23 @@ test_that("Warns if encoding is misspecified", {
   expect_equal(the_lint$message, "Invalid multibyte string. Is the encoding correct?")
   expect_equal(the_lint$line_number, 1L)
 })
+
+test_that("Can extract line number from parser errors", {
+  skip_if_not(getRversion() >= "4.0")
+
+  with_content_to_parse("\"ok\"\nR\"---a---\"", {
+    expect_equal(error$message, "Malformed raw string literal.")
```

Comment:
I'd prefer using ' to wrap this string for readability. using a newline over \n may be preferable as well...

Reply:
Done using `trim_some()` for better readability.